R/utils.R

```diff
@@ -35,22 +47,29 @@ reformat_data <- function(data){
       data["granularity"] <- "day"
     }
 
-    data$date <- as.Date(paste(data$year, data$month, data$day, sep = "-"))
+    data$date <- as.POSIXct(paste(data$year, data$month, data$day, sep = "-"))
     data$rank <- as.numeric(data$rank)
-
   } else {
-    data <- data.frame(do.call(rbind, data), stringsAsFactors = FALSE)
-
-    data$date <- as.Date(substr(data$timestamp, 1, 8), format = "%Y%m%d")
+    data <- as.data.frame(do.call(rbind, data), stringsAsFactors = FALSE)
+    data$date <- as.POSIXct(as.character(data$timestamp), format = "%Y%m%d%H", tz = "GMT")
```

Comment:
Agreed. Are there any API calls that generate anything more granular than daily right now?